Tests/MeiliSearchIntegrationTests/DocumentsTests.swift

```diff
@@ -127,6 +127,22 @@ class DocumentsTests: XCTestCase {
         }
         self.wait(for: [getExpectation], timeout: 3.0)
     }
+    
+    func testGetOneDocumentWithIntIdentifierAndFail() {
+        let getExpectation = XCTestExpectation(description: "Get one document and fail")
+        self.client.getDocument(
+            UID: uid,
+            identifier: 123456
+        ) { (result: Result<Movie, Swift.Error>) in
+            switch result {
+            case .success:
+                XCTFail("Document has been found while it should not have")
+            case .failure:
+                getExpectation.fulfill()
+            }
+        }
+        self.wait(for: [getExpectation], timeout: 3.0)
+    }
```

Comment:
This tests is useless since it does not check if the error is due to a 404 or due to an internal error in the code. Could you change it with the following ?

```suggestion
    func testGetOneDocumentWithIntIdentifierAndSucceed() {
        let movie: Movie = Movie(id: 10, title: "test", comment: "test movie")
        let getExpectation = XCTestExpectation(description: "Get one document using and int identifier and succeed")
        self.client.getDocument(
                   UID: uid,
                   identifier: 10
               ) { (result: Result<Movie, Swift.Error>) in

                   switch result {
                   case .success(let returnedMovie):
                       XCTAssertEqual(movie, returnedMovie)
                   case .failure(let error):
                       XCTFail(error.localizedDescription)
                   }
                   expectation.fulfill()

               }
    }
```